web/client/configs/localConfig.json

```diff
@@ -897,7 +897,33 @@
           "Attribution",
           "Home",
           {
-            "name": "GeoStory"
+            "name": "GeoStory",
+            "cfg": {
+                "fontFamilies": [
+                    {
+                        "family": "Arial"
+                    },
+                    {
+                        "family": "Georgia"
+                    },
+                    {
+                        "family": "Impact"
+                    },
+                    {
+                        "family": "Tahoma"
+                    },
+                    {
+                        "family": "Times New Roman"
+                    },
+                    {
+                        "family": "Titillium Web",
```

Comment:
@mumairr I don't think we want to include Trillium in MapStore. Are you sure about this hardcoded list?

Reply:
@giohappy no we don't and we shouldn't. It was merely to support the idea of changes requested in the PR.
Should I remove this?

Reply:
@mumairr yes please remove those font in the default configuration, just mention in the PR description to add this to properly test the fixes, thanks

Reply:
The font configuration is still here, please restore the localConfig